distribution/springboot_setup/receiver/application.properties

```diff
@@ -8,17 +8,17 @@ spring.activemq.password=password
 
 # Configuration of the openMRS database to read from
 # Driver class of the openMRS datasource (should not be changed in a usual use)
-spring.openmrs-datasource.driverClassName=com.mysql.jdbc.Driver
+spring.openmrs-datasource.driverClassName=com.mysql.cj.jdbc.Driver
 # Dialect of the openMRS datasource (should not be changed in a usual use)
 spring.openmrs-datasource.dialect=org.hibernate.dialect.MySQLDialect
 # DON'T CHANGE THIS PROPERTY
 spring.openmrs-datasource.ddlAuto=none
 # Url of the openMRS datasource
 spring.openmrs-datasource.jdbcUrl=jdbc:mysql://localhost:3306/openmrs
 # User name of the openMRS datasource
```

Comment:
Let's rename to "# Privileged user that has access to the OpenMRS datasource (likely 'root')"